bootstrap.sh

```diff
@@ -0,0 +1,18 @@
+#! /usr/bin/env bash
+
+export PKG_CONFIG_PATH="/usr/lib/pkgconfig/;/lib/pkgconfig/;/usr/share/pkgconfig/"
+
+# Delete existing buildfiles
+#---------------------------------------------------
+rm -rf build
+
+
+# Call meson
+#---------------------------------------------------
+meson setup build -Dprefix="/usr"
+
+
+# Build nix
+#---------------------------------------------------
+cd build
+ninja
```

Comment:
Fix that alerts.

Reply:
what does this change?

Reply:
GitHub shows a red minus signal indicating that the last line is not blank.